tools/torchci/clickhouse_query_perf.py

```diff
@@ -191,11 +268,35 @@ def results_compare(args: argparse.Namespace) -> None:
     if not results_folder.exists():
         results_folder.mkdir(parents=True)
     print(
-        f"Comparing results for query: {args.query}\nNum tests: {len(tests)}\nHead: {args.head} Base: {args.base}"
+        f"Comparing results for query: {args.query}\nNum tests: {len(tests)}\nHead: {args.head}\n Base: {args.base}"
```

Comment:
Let's maybe rename Base? Confusing to see now an output like:
```
Head: unstaged
 Base: HEAD
```

Something like: reference and new? reference and current?

Reply:
need to think about those, as they match the argument names